internal/nhooyr.io/websocket/write.go

```diff
@@ -236,6 +236,13 @@ func (mw *msgWriter) close() {
 
 func (c *Conn) writeControl(ctx context.Context, opcode opcode, p []byte) error {
 	ctx, cancel := context.WithTimeout(ctx, time.Second*5)
+	switch opcode {
+	case opPong:
+		if c.pingTimeout > 0 {
+			ctx, cancel = context.WithTimeout(ctx, c.pingTimeout)
+		}
+	}
```

Comment:
@schmidtw does this complete #88 as well? `disconnect & reconnect websocket when ping request is not received in the allowed time`

or were we talking about `pingInterval` triggering a disconnect & reconnect as well?